src/db/migrations/20230703010356_add-profile-table.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export function up(knex) {
+  return knex.schema
+    .createTable('profiles', table => {
```

Comment:
Let me make a prediction that someone's going to get confused in the future if we don't rename this :sweat_smile: 

```suggestion
    .createTable('onerep_profiles', table => {
```

Reply:
I'm not against it, I chose to not do it because it's data that's submitted directly to us, and it may not always be strictly limited to this... we can change it later, but there's a cost to migrating the data.